src/index.js

```diff
@@ -98,10 +172,45 @@ function listTopMovies(input) {
         .addClass('.error')
         .text(messages[Error.NO_RESULTS])
         .fadeIn('slow');
+      delayFlashHide();
     });
 }
 
-function fetchMovies(year) {
-  let url = urlJoin(proxyUrl, baseUrl, '/year/', year);
-  return fetchDocument(url).then(doc => parseMovies(doc));
+/* infinite scroll */
+
+$window.scroll(e => {
+  if ($(e.target).scrollTop()) {
+    $arrowToTop.fadeIn();
+  } else {
+    $arrowToTop.fadeOut();
+  }
```

Comment:
>arrow gets rerendered

What do you mean by this? What kind of events you are talking about, scroll ones? 😕